routes/submitForm.js

```diff
@@ -1,16 +1,63 @@
-/*************************
-TODO PART 1
-TODO find a module that can submit mysql requests
-TODO write a query request in the function below
-TODO make it transaction based
-
-TODO Part2
-TODO put the config info for the mysql server in the config
-TODO send the front end a success message on success and an error message with error code on fail
-TODO remove these comments!
-*************************/
+var mysql = require("mysql");
+var config = require("konfig")();
 
 var router = function(req, res) {
+    var connection = mysql.createConnection({
+        host: config.app.mysql.host,
+        user: config.app.mysql.user,
+        password: config.app.mysql.password
+    });
+
+    connection.connect(function(err) {
+        if (err) {
+            console.error(err.stack);
+            res.send("Connection Error: " + err.code);
+        } else {
+            connection.beginTransaction(function(err) {
+                if (err) {
+                    console.error(err.stack);
+                    res.send("Begin Transaction Error: " + err.code);
+                } else {
+                    connection.query("USE test", function(err, result) {
+                        if (err) {
+                            connection.rollback(function() {
+                                console.error(err.stack);
+                                res.send("Rollback Error: " + err.code);
+                            });
+                        } else {
+                            connection.query("CREATE TABLE IF NOT EXISTS names (id int, name varchar(30))", function(err, result) {
+                                if (err) {
+                                    connection.rollback(function() {
+                                        console.error(err.stack);
+                                        res.send("Rollback Error: " + err.code);
+                                    });
+                                } else {
+                                    connection.query("INSERT INTO names VALUES (2112, 'Tom Sawyer')", function(err, result) {
```

Comment:
We want to make sure we are inserting the values given to us in req.params, so that the front end can define what information is added.